analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -297,7 +305,5 @@ private static ProgramState InitLocals(ControlFlowBranch branch, ProgramState st
     private static bool IsReachable(ControlFlowRegion region, ExceptionState thrown) =>
```

Comment:
This method was unnecessarily complex. If the Exception is not `Unknown`, the `DerivesFrom` check will catch the `System_Exception`- and the `System_Object`-case.

Reply:
Thanks for the hint. I just asked myself the same thing and this saved me investigation/question.